package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "renpool",
+  "version": "0.1.0",
+  "repository": "git@github.com:Ethernautas/renpool.git",
+  "author": "Fede Rodes",
+  "license": "MIT",
+  "scripts": {
+    "build": "hardhat compile",
+    "test": "hardhat test",
+    "test:mainnet": "FORK=mainnet hardhat test",
```

Comment:
@fede-rodes here's how I solve the config issue. The `FORK` env var indicates where to fork from when running tests.

Check `hardhat.config.js` for details.

Reply:
Perfect!